### PR TITLE
Python log injection traceID/spanID

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

- General formatting
- Added new file based on customer feedback - docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+description: Learn how to configure traceId, spanId, and trace flags data injection into user logs in Python applications.
+---
+
+Here's how to add traceId and spanId into logs for Python lambda. You have to get a context from current span and attach it to the log.
+
+## Python lambda
+
+1. Add library import:
+  ```py
+  from OpenTelemetry import trace
```
Comment (pbrzoska-sumo) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md:14]: no capitals in the code please

Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md:14]:
```suggestion
  from opentelemetry import trace
```

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+id: traceid-spanid-injection-into-logs
+title: Python TraceId and SpanId injection into logs configuration
+sidebar_label: TraceId and SpanId injection into logs
+description: Learn how to configure traceId, spanId, and trace flags data injection into user logs in Python applications.
+---
+
+Here's how to add traceId and spanId into logs for Python lambda. You have to get a context from current span and attach it to the log.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md:8]:
```suggestion
Here's how to add `trace_id` and `span_id` into your Python lambda logs. You have to get a context from current span and attach it to the log.

:::important
Make sure that the code from below is active in the `.py` file where your application requests are traced with OpenTelemetry python instrumentation. Logs not related to traced requests will not have `span_id` or `trace_id` filled in.
:::
```

Comment (kimsauce) [reply docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md:8]: @pbrzoska-sumo , does this capture your change correctly? ☝️

**Review by stacykor — APPROVED** [review]
LGTM

File: docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+id: traceid-spanid-injection-into-logs
+title: Python TraceId and SpanId injection into logs configuration
+sidebar_label: TraceId and SpanId injection into logs
+description: Learn how to configure traceId, spanId, and trace flags data injection into user logs in Python applications.
+---
+
+Here's how to add `trace_id` and `span_id` into your Python lambda logs. You have to get a context from current span and attach it to the log.
```
Comment (kimsauce) [inline docs/apm/traces/get-started-transaction-tracing/opentelemetry-instrumentation/python/traceid-spanid-injection-into-logs.md:8]:
```suggestion
To add `trace_id` and `span_id` to your Python lambda logs, you'll have to get a context from current span and attach it to the log.
```

**Pull request merged**